tests/utils/set_params.cpp

```diff
@@ -130,11 +134,11 @@ void setInvertParam(QudaInvertParam &inv_param)
   if (kappa == -1.0) {
     inv_param.mass = mass;
     inv_param.kappa = 1.0 / (2.0 * (1 + 3 / anisotropy + mass));
-    if (dslash_type == QUDA_LAPLACE_DSLASH) inv_param.kappa = 1.0 / (8 + mass);
+    if (dslash_type == QUDA_LAPLACE_DSLASH) inv_param.kappa = 1.0 / ((laplace3D == 3 ? 6 : 8) + mass);
```

Comment:
I saw elsewhere in the code that `(laplace3D == 3)` was replaced with something more like `(laplace3D < 4)` or `(laplace3D != -1)` or something like that, I forget the convention; I preferred that to the hard compare against `3`, and it seems safer.

Reply:
The less than convention was there to give the option for future X/Y/Z orthogonal dimension.  Good to make it consistent though.

Reply:
8fb71814ce06ce0a626b6a7c3ec7deae024e054a